modules/custom/d_content/modules/d_content_init/src/ContentInitManagerBlock.php

```diff
@@ -92,14 +98,11 @@ protected function createBlockPlugin(array $block) {
       $values['id'] = $values['id'] ?? 'block_plugin_' . str_replace('-', '_', $this->uuid->generate());
       $this->getCurrentThemeIfNotDefined($values);
       return $this->saveEntity('block', $values);
-    }
-    catch (PluginNotFoundException $e) {
+    } catch (PluginNotFoundException $e) {
```

Comment:
This is not proper for Drupal code standards https://www.drupal.org/docs/develop/coding-standards/php-exceptions.

Please also fix remaining `catch` clauses in this file